src/ApiPort.VisualStudio/ApiPort.VisualStudio.csproj

```diff
@@ -31,7 +31,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <Prefer32Bit>false</Prefer32Bit>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
I have it on my backlog to add a CI build to this.  We can leave it for now. When I enable a CI build, I'll make the TreatWarningsAsErrors consistent in the repo.